src/CommonITILSatisfaction.php

```diff
@@ -246,16 +246,16 @@ public function post_UpdateItem($history = 1)
     /**
      * display satisfaction value
      *
-     * @param int $value Between 0 and 5
+     * @param int $value Between 0 and 10
      **/
     public static function displaySatisfaction($value)
     {
 
         if ($value < 0) {
             $value = 0;
         }
-        if ($value > 5) {
-            $value = 5;
+        if ($value > 10) {
+            $value = 10;
         }
 
         $out = "<div class='rateit' data-rateit-value='$value' data-rateit-ispreset='true'
```

Comment:
You have to increase max value in form (in `$('#stars').rateit()` call).

Reply:
already done =)